rust/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 
-members = ["indexer-metrics", "moving-average", "processor", "server-framework"]
+members = ["indexer-metrics", "moving-average", "processor", "server-framework", "transaction-filter"]
```

Comment:
Maybe we wanna do `aptos-transaction-filter`? We tend to prefix all new crates with `aptos-`, the crates in this repo are a bit of an exception.